### DISCUSSION TITLE
CSGO Demo Manager 2.13.15 -> CS Demo Manager 3.4.0

### DISCUSSION THREAD

**Discussion by mavproductions** [akiver/cs-demo-manager#804] [Ideas]
Hey @akiver and all,
During CSGO, I used CSGO Demo Manager a bit, but I was worried about updating as I relied heavily on it's metadata tracking of my demo comments, and other data for collecting long term demos for frag-movie ideas.

I did a few updates from `CSGO:DM 2.13.15` and when I imported the data back, I remember running into issues of missing or corrupt demos/data. I can try reproducing again if that would help.

**IDEALLY**

I would rather fast-track and migrate this CSGO demo metadata to `CSDM 3.4.0`. Is this possible in the slightest? What would be my best chance at keeping integrity of all my CSGO demo data, and not messing up `CSDM 3.4.0`? I know I can always backup before I make changes and do trial-and-error, but I'd prefer some experienced insight.

Appreciate y'all

Cheers,

**Comment by akiver** [comment]
Hi!

I think it *should* be possible to migrate the V2 data (comment and status) to V3, I will give it a try.

**Reply by mavproductions** [reply]
Thanks man. Even going from 2.13.15 to other nearby versions I remember having issues, so if it does go to V3 that would be incredible. I don't mind sharing my DB if that helps you. Just ping me and I can email it to you or something if need be.

**Comment by akiver** [comment]
Available as of v3.5.0
